modules/oc-mirror-oci-format.adoc

```diff
@@ -73,12 +73,10 @@ mirror:
 [source,terminal]
 ----
 $ oc mirror --config=./imageset-config.yaml \ <1>
-  --include-local-oci-catalogs                <2>
-  docker://registry.example:5000              <3>
+  docker://registry.example:5000              <2>
 ----
 <1> Pass in the image set configuration file. This procedure assumes that it is named `imageset-config.yaml`.
-<2> Use the `--include-local-oci-catalogs` flag to enable mirroring local OCI catalogs along with other remote content.
-<3> Specify the registry to mirror the content to. The registry must start with `docker://`. If you specify a top-level namespace for the mirror registry, you must also use this same namespace on subsequent executions.
+<2> Specify the registry to mirror the content to. The registry must start with `docker://`. If you specify a top-level namespace for the mirror registry, you must also use this same namespace on subsequent executions.
```

Comment:
@subhtk I think registry should be mentioned as below in the imageSetConfig.yaml file and not as  docker:// if i am not wrong. 

the way we include local oci in the imagesetconfig.yaml - with oci://<path-to-local-catalog>